packages/AdobeServer/dist/AdobeServerSideKit.iife.js

```diff
@@ -46,6 +46,11 @@ var mParticleAdobe = (function () {
         this.common = common || {};
     }
     EventHandler.prototype.logEvent = function(event) {
+        var customAttributes = {};
```

Comment:
should this instead be common.js instead of iife?

media is done only through npm, so i think it needs to be the common.js file. 

we can discuss further monday as i should review how adobe is all structured also.  i have notes somewhere, about this.  and if i can't find them i can figure it out.  also what about adobe client side files?